DAOBOX ENF Grant Application

eosdaobox opened the pull request:
### Project Abstract

> Please provide a brief description of your project here summarising key points (1-2 paragraphs).
>
> If your application is a follow-up to a previous grant, please mention which one in the first line of the abstract and include a link to previous pull requests if applicable.
> Please delete these lines that start with a `>`

### For which grant [level](https://github.com/eosnetworkfoundation/grant-framework#grant-levels) are you applying?

- [ ] **Level 1**:  Up to  $10,000, 2 approvals
- [ ] **Level 2**:  Up to  $50,000, 3 approvals
- [x] **Level 3**:  Up to $200,000, 4 approvals

### Application Checklist

- [x] The [application template]([xxxx](https://github.com/eosnetworkfoundation/grant-framework/blob/main/applications/application-template.md)) has been copied, renamed (`my_project_name.md`) and updated.
- [x] The application file (`my_project_name.md`) has no spaces in its name, ends with the `.md` extension has been placed in the `applications` directory.
- [x] I have read and understood the:
  - [ ] [FAQs](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/faq.md), 
  - [ ] [application guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/grant_guidelines_per_category.md) and 
  - [ ] [announcement guidelines](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/announcement-guidelines.md).
- [x] An EOS address for the payment of the milestones is provided inside the application.
- [x] I have read and acknowledge the [terms and conditions](https://github.com/eosnetworkfoundation/grant-framework/blob/main/docs/T&Cs.md).
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] This project has NOT been part of a token sale or NFT sale.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/eosnetworkfoundation/grant-milestones#process).

### How Did You Hear About our grants program?

- [x] Social Media
- [ ] EOSNetwork.com
- [x] Personal Recommendation
- [ ] ENF Fireside Chat
- [ ] Investor/VC
- [ ] Online Search
- [ ] Hackathon
- [ ] Other: _______

eosdaobox commented:
Dear @tedcahalleos 
How are you? This is DAOBOX. DAOBOX is a DAO project invested by Defibox. Raven from Defibox told me about this ENF Grant Application and we are applying for it. We are new to it and we got some advice from them already. So, this is the application, thanks if you could check. Since our team is invested and aided by them, you can always reach me via my E-mail stated in the .md file, or just contact Raven at Telegram.

Thank you very much for your help.

DAOBOX team

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/eosnetworkfoundation/grant-framework?pullRequest=136) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/eosnetworkfoundation/grant-framework?pullRequest=136) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/eosnetworkfoundation/grant-framework?pullRequest=136) it.</sub>

eosdaobox commented:
> Hi, There are three issues with this grant that must be corrected before we can process.
> 
> 1. You must remove words added to the Total Cost line as it breaks the automatic parser (see app template)
> 2. Grants now have a maximum value of $200,000 USD
> 3. We need a person name for the Grant.  We cannot accept DAOBOX.  If you would like to contact us and have us hold that privately, please email [grants@eosnetwork.com](mailto:grants@eosnetwork.com) and we will discuss with you.

Hello, Ted. Thanks for your feedback. Please check again, we have improved based on your feedback
1. We deleted already
2. we changed to total amount to 196,000 USD
3. Already changed the name

Again thanks for your help. If there is anything, you can always leave me a message. Thanks.

ABurton22 commented:
Grant request has been rejected by Grant Committee. Closing out pull request.

Thank you,
Amy Burton
